cuba.yml

```diff
@@ -545,3 +545,23 @@ CUBA_KEYS:
     definition: Classification of a fluid flow type
     shape: [1]
     type: integer
+
+  VECTOR:
+    definition: A vector
+    shape: [1]
+    type: double
+
+  POINT:
+    definition: A point in a 3D space system
+    shape: [3]
+    type: double
+
+  FACE:
+    definition: A face
+    shape: [1]
+    type: double
```

Comment:
Good point! a face in a mesh is defined by the uuid of the nodes. The nodes can be used to define the basis vectors if additional information on the "order" of the nodes, i.e., how to map them to vectors is given (say first node is the origin, second defines the first vector, and so on). If we add this information to the basic definition of a cell, or alternatively define Box based on cell with additional attributes then we can assume that essentially a box is a special mesh! This should be added to the specifications!